images/access-setup/content/bin/setup_compute.sh

```diff
@@ -142,6 +142,8 @@ generate_shared_manifests(){
   printf -- "- Generating shared manifests:\n"
   printf -- "  - tekton-chains manifest:\n"
   tekton_chains_manifest 2>&1 | indent 4
+  printf -- "  - tekton-results manifest:\n"
+  tekton_results_manifest
```

Comment:
This had to be removed so that the user could be asked for credentials for configuring the database. If this is left in, the script hangs with no output until the credentials are entered and then the output is displayed which would be too late.

Reply:
Ok I would have preferred to keep the possibility to run ckcp script in a non interactive way so we can just run it and forget about it. But we can handle this in a later task.

Reply:
There should be no user interaction when using the script (both for UX, and because it's used by the CI bot). Please find a way to inject the credentials.

Reply:
Actually I am wondering if this is going to work with github test
@Roming22 @xinredhat if you confirm that github-ci is running a fresh ckcp cluster then the script is going to hang for user input.

Reply:
I don't know if this makes sense, but can we provide credentials using config.yaml?

Reply:
I've updated to now pass tekton details as arguments or generate if not passed into the scripts